Doc/library/importlib.rst

```diff
@@ -775,6 +775,131 @@ ABC hierarchy::
         itself does not end in ``__init__``.
 
 
+:mod:`importlib.resources` -- Resources
+---------------------------------------
+
+.. module:: importlib.resources
+    :synopsis: Package resource reading, opening, and access
```

Comment:
Block contents in reST directives should start with three spaces, not four.

Reply:
Seems like we have some inconsistency in this file.  I just copied it from every other section in that file except the top level `.. module:: importlib`.